testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_custom_conflicts.py

```diff
@@ -0,0 +1,1013 @@
+    completed = replicator.getCompleted(repl)
+    replicator.stop(repl)
+    assert total == completed, "total is not equal to completed"
+
+    sg_docs = sg_client.get_all_docs(url=sg_url, db=sg_db, auth=session)["rows"]
+
+    # creating conflict for docs on SG
+    sg_client.update_docs(url=sg_url, db=sg_db, docs=sg_docs, number_updates=2,
+                          property_updater=property_updater, auth=session)
+
+    # creating conflict for docs on CBL
+    doc_ids = db.getDocIds(cbl_db)
+    cbl_docs = db.getDocuments(cbl_db, doc_ids)
+    for doc_id in cbl_docs:
+        for _ in range(2):
+            log_info("Updating CBL Doc - {}".format(doc_id))
+            data = cbl_docs[doc_id]
+            data = property_updater(data)
+            data["cbl_random"] = random_string(length=10, printable=True)
+            db.updateDocument(cbl_db, doc_id=doc_id, data=data)
+
+    repl_config = replicator.configure(cbl_db, sg_blip_url, continuous=True, channels=channels,
+                                       replicator_authenticator=replicator_authenticator,
+                                       replication_type="pull", conflict_resolver="delayed_local_win")
+    repl = replicator.create(repl_config)
+    replicator.start(repl)
+    sleep(5)  # sleeping so that replicator is in Conflict resolver
+    log_info("Stopping Replicator before CCR finishes - 10 sec delay for CCR")
+    replicator.stop(repl, max_times=100)
+    total = replicator.getTotal(repl)
+    completed = replicator.getCompleted(repl)
+    assert total == completed, "total is not equal to completed"
+
+    # printing doc content before replication conflicted docs
+    sg_docs_content = sg_client.get_bulk_docs(sg_url, sg_db, doc_ids, session)[0]
+    cbl_docs = db.getDocuments(cbl_db, doc_ids)
+    for sg_doc in sg_docs_content:
+        doc_id = sg_doc["_id"]
+        cbl_doc = cbl_docs[doc_id]
+        assert sg_doc["sg_new_update1"] != cbl_doc["sg_new_update1"], "Replication stopped before resolving" \
+                                                                      " the conflict"
+        assert sg_doc["sg_new_update2"] != cbl_doc["sg_new_update2"], "Replication stopped before resolving" \
+                                                                      " the conflict"
+        assert sg_doc["sg_new_update3"] != cbl_doc["sg_new_update3"], "Replication stopped before resolving" \
+                                                                      " the conflict"
+        assert "random" not in cbl_doc, "Replication stopped before resolving the conflict"
+        assert "cbl_random" not in sg_doc, "Replication stopped before resolving the conflict"
+
+
+@pytest.mark.listener
+@pytest.mark.custom_conflict
+@pytest.mark.replication
+@pytest.mark.parametrize("replicator_type", [
+    "pull",
+    "push_pull"
+])
+def test_delete_not_wins_custom_conflicts(params_from_base_test_setup, replicator_type):
+    """
+    @summary: resolve conflicts as per local doc
```

Comment:
please add summary as written in spreadsheet

Reply:
see previous comments